tests/testtypes/consensus-spec-tests/types_basics.go

```diff
@@ -7,12 +7,25 @@ package consensus_spec_tests
 import "github.com/prysmaticlabs/go-bitfield"
 
 //go:generate go run -cover ../../../cmd/sszgen -type SingleFieldTestStruct -out gen_single_field_test_struct_ssz.go
+//go:generate go run -cover ../../../cmd/sszgen -type SmallTestStruct -out gen_small_test_struct_ssz.go
+//go:generate go run -cover ../../../cmd/sszgen -type FixedTestStruct -out gen_fixed_test_struct_ssz.go
 //go:generate go run -cover ../../../cmd/sszgen -type BitsStruct -out gen_bits_struct_ssz.go
 
 type SingleFieldTestStruct struct {
 	A byte
 }
 
+type SmallTestStruct struct {
+	A [1]byte `ssz-size:"2" ssz:"bits"`
+	B [1]byte `ssz-size:"2" ssz:"bits"`
+}
+
+type FixedTestStruct struct {
+	A [1]byte `ssz-size:"1" ssz:"bits"`
+	B [1]byte `ssz-size:"8" ssz:"bits"`
+	C [1]byte `ssz-size:"4" ssz:"bits"`
+}
```

Comment:
Ah, no no 

The types of the fields are

```
class SmallTestStruct(Container):
    A: uint16
    B: uint16


class FixedTestStruct(Container):
    A: uint8
    B: uint64
    C: uint32
```

I.e. 

```go
type SmallTestStruct struct {
	A uint16
	B uint16
}

type FixedTestStruct struct {
	A uint8
	B uint64
	C uint32
}
```

Reply:
You don't need any struct tags on there. Those are more for the weird types :D

Reply:
i tried these but they dont generate tests when i ran `go generate` with errors 

`/home/lavishq/ssz/tests/testtypes/consensus-spec-tests/gen_fixed_test_struct_ssz.go:15:2: T (type [8]byte) does not satisfy ssz.commonBitsLengths`
`/home/lavishq/ssz/tests/testtypes/consensus-spec-tests/gen_fixed_test_struct_ssz.go:16:2: T (type [4]byte) does not satisfy ssz.commonBitsLengths`

so i tinkered to remove the errors^
would fix asap

Reply:
Ah, damn, didn't see that there's a list of uint16 in there in VarTestStruct. Yeah, just drop that type and the ComplexTestStruct. List of primitives needs a lot more work there so it's very out of scope for this PR. Just leave SmallTestStruct and FixedTestStruct and shoult be fine like that.

Reply:
Also pls add the test call for the two types here https://github.com/karalabe/ssz/blob/main/tests/consensus_specs_test.go#L51 

That should make this PR complete :)